lib/test/t_HSICStat_std.cxx

```diff
@@ -85,7 +85,9 @@ int main(int, char *[])
       Sample test = X.getMarginal(i);
       /* Set input covariance scale */
       Cov1.setScale(test.computeStandardDeviation());
-      Scalar hsicIndex = estimatorTypeV.computeHSICIndex(test, Y, Cov1, Cov2, W);
+	  CovarianceMatrix CovMat1(Cov1.discretize(test));
```

Comment:
There are a few cosmetic errors here: the blanks are too long and the variable names start with an uppercase letter instead of a lowercase letter.